### PR TITLE
Handle time.Time params in command line.

### PR DESCRIPTION
**Pull request by arangamani**

### CODE REVIEW

File: rsapi/commands.go [hunk 1]
```diff
@@ -104,6 +105,12 @@ func (a *API) ParseCommand(cmd, hrefPrefix string, values ActionCommands) (*Pars
 					name, value)
 			}
 			*coerced = append(*coerced, APIParams{name: val})
+		case "*time.Time":
+			val, err := time.Parse(time.RFC3339, value)
```
Comment (ryanoleary) [inline rsapi/commands.go]: Does this mean all times that are used by RSC must be RFC3339 type? I know our APIs use various **different** time formats, so how does that work with this?

Comment (arangamani) [inline rsapi/commands.go]: There is one other time type and that is called `RubyTime`. I only fixed the SS issue.

Comment (arangamani) [inline rsapi/commands.go]: Looks like only CA and SS APIs accept `time.Time` attribute type and I believe they both will be okay with RFC3339. The CM15 API seems to use the `string` type for parameters such as `start_date` in audit entries query.

**Comment by raphael** [comment]
LGTM

File: rsapi/commands.go (hunk 1 above)
Comment (tve) [inline rsapi/commands.go]: Would be really nice to support a Unix timestamp integer (seconds since epoch)...

**Comment by arangamani** [comment]
@tve PR updated.

**Pull request merged**